go/firewallsso/base.go

```diff
@@ -242,7 +243,7 @@ func (fw *FirewallSSO) logger(ctx context.Context) log15.Logger {
 }
 
 func (fw *FirewallSSO) getDst(ctx context.Context, proto string, toIP string, toPort string) string {
-	if cc := connector.ConnectorsContainerFromContext(ctx); cc != nil {
+	if fw.UseConnector == "enabled", cc := connector.ConnectorsContainerFromContext(ctx); cc != nil {
```

Comment:
Small detail, you can use `sharedutils.IsEnabled(fw.UseConnector)` for a cleaner handling in case the value is set to `yes` or `1`

Reply:
Done!
Thanks!